Jenkinsfile

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env groovy
+
+/**
+ * Build openwrt project
+ *
+ * Configure as needed, pull in the feeds and run the build. Allows
+ * overridding feeds listed using build parameters.
+ * 
+ * Note this script needs some extra Jenkins permissions to be
+ * approved.
+ */
+// TODO be careful with using dir() until JENKINS-33510 is fixed
+
+def customFeeds = [
+    ['packages', 'packages', 'https://github.com/CreatorDev'],
+    ['ci40', 'ci40-platform-feed', 'https://github.com/CreatorDev'],
+    ['creator', 'creator-feed', 'https://github.com/CreatorDev'],
+]
+def feedParams = []
+for (feed in customFeeds) {
+    feedParams.add(string(defaultValue: '', description: 'Branch/commmit/PR to override feed. \
+        (Can be a PR using PR-< id >)', name: "OVERRIDE_${feed[1].toUpperCase()}"))
+}
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '5')),
+    parameters([
+        booleanParam(defaultValue: false,
+            description: 'Build extra tools such as toolchain, SDK and Image builder',
+            name: 'BUILD_TOOLS'),
+        booleanParam(defaultValue: false, description: 'Build *all* packages for opkg',
+            name: 'ALL_PACKAGES'),
+        stringParam(defaultValue: 'target/linux/pistachio/creator-platform-default-cascoda.config',
```

Comment:
this .config file is not present in upstream-pistachio-support branch, so the default value either needs to be changed, or this parameter needs to be removed entirely, as anyway it doesn't appear to be used (useful) in this Jenkinsfile